server/resources/js/app/widb.js

```diff
@@ -88,9 +96,11 @@ class WirelessDebug {
     let messageData = JSON.parse(message.data);
 
     if (messageData.messageType === 'logData') {
-      this.data_table.destroy();
-      this.logTable_.append(messageData.logEntries);
-      this.data_table = $('#log-table').DataTable();
+      // If we get more log data, append the log data to the table and scroll to
+      // the bottom of the table.
+      this.data_table.row.add($(messageData.logEntries)).draw();
+      let scrollBody = $('.dataTables_scrollBody');
```

Comment:
Can you make it consistent what you are using in terms of camelCase vs _ seperated. Style guide says to use camelCase http://google.github.io/styleguide/javascriptguide.xml#Naming